conf/map/script.conf

```diff
@@ -51,13 +51,13 @@ script_configuration: {
 
 	// Default value of the 'min' argument of the script command 'input'.
 	// When the 'min' argument isn't provided, this value is used instead.
-	// Defaults to 0.
+	// Defaults to 0
 	input_min_value: 0
 
 	// Default value of the 'max' argument of the script command 'input'.
 	// When the 'max' argument isn't provided, this value is used instead.
-	// Defaults to INT_MAX.
-	//input_max_value: 2147483647
+	// Max allowed value that can be used is INT_MAX (2147483647).
+	// Defaults to 10000000
```

Comment:
Here it was actually true that the default was `INT_MAX` and not `10000000`. There's a discrepancy between the value we set in the default config file and the default value set in source (which takes effect when you comment it out from the config file).

Please either mention it in the comment here, or change the default in script.c (and in that case, I'll mention it in the release notes)

Reply:
done